src/LiquidityPool.sol

```diff
@@ -462,10 +464,38 @@ contract LiquidityPool is Initializable, OwnableUpgradeable, UUPSUpgradeable, IL
         emit Rebase(getTotalPooledEther(), eETH.totalShares());
     }
 
+    function payProtocolFees(uint128 _protocolFees) public {
+        if (msg.sender != address(etherFiAdminContract) && msg.sender != address(this)) revert IncorrectCaller();
+        totalValueOutOfLp = totalValueOutOfLp + _protocolFees;
+        uint256 treasuryShares = sharesForAmount(_protocolFees);
+        eETH.mintShares(treasury, treasuryShares);
+    }
+
+    //one time function to mint shares when we change split to 0
+    //need to 
+    function mintShareOnChangeSplit(uint256[] memory _validatorIds, uint256[] memory _beaconBalances) external onlyAdmin { 
+            uint256 totalAccruedRewards = 0;
+            for (uint256 i = 0; i < _validatorIds.length; i++) {
+                require(_beaconBalances[i] <= 33 ether, "Invalid Beacon Balance");
+                (uint256 treasury, uint256 nodeOperator, uint256 tnft, uint256 bnft) = nodesManager.calculateTVL(_validatorIds[i], _beaconBalances[i]);
```

Comment:
I believe calculateTVL is currently broken in the current implementation, so we need to be very careful using it here to handle payouts (It does not properly account for funds in eigenlayer `delegationManager` withdrawal queue

Reply:
how do we update it so it is correct?

Reply:
<img width="1269" alt="image" src="https://github.com/user-attachments/assets/3fb43fc1-e385-4476-bb9b-d6c431c845ac">